### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by FilipeMarch** [qtilities/magnifiqus#5]
We are missing USAGE topic on README.md
I don't know how to use the tool

**Comment by redtide** [comment]
Seems quite trivial to me: once run the application you get a system tray icon. Clicking on it will display a movable, undecorated window that magnify what is below the mouse cursor around the screen, unless you are on Wayland (or any other OS except X11 based system) where it doesn't work, or maybe you are not using any dock/taskbar panel? Which system are you using?

**Comment by FilipeMarch** [comment]
All right, thanks. I see it now. I'm using Arch Linux.

Although it looks trivial for most people how to use this tool, when I opened it for the first time nothing happened. I was looking around, "where is it?", and I didn't see nothing. Sure, I was not paying attention to the `system tray` icons, normally my dock appears only when I hover the mouse over the top of the screen. Well, I tried to open the application 2 or 3 times, and I didn't see anything happening at all, I concluded it was broken or unresponsive.

If I had seen "this is a system tray application" on README I would have paid attention to the dock. I believe it would not hurt to make this statement more obvious.

Thanks for you attention! `magnifiqus` works nicely.
